react-app/src/services/food-trust.ts

```diff
@@ -0,0 +1,89 @@
+// import { isConstructorDeclaration } from "typescript";
+
+export class FoodTrustAPI {
+    APIBase = "/api/foodtrust/";
+
+    async foodTrustAuthToken() {
```

Comment:
This is another function that should be removed since the front end doesn't know about the token now.

Reply:
Done